packages/wrangler/src/vectorize/insert.ts

```diff
@@ -16,11 +21,17 @@ export function options(yargs: CommonYargsArgv) {
 			description: "The name of the Vectorize index.",
 		})
 		.options({
-			vectors: {
-				type: "array",
+			file: {
+				describe:
+					"A file containing line separated json (ndjson) vector objects.",
 				demandOption: true,
+				type: "string",
+			},
+			"batch-size": {
 				describe:
-					"An array of one or more vectors in JSON format to insert into the index",
+					"Number of vector records to include when sending to the Cloudflare API.",
```

Comment:
Should there be a max?

Reply:
I think the backend should be the one worrying about this (since clients could edit these file values to whatever they want anyway) but I went ahead and put a soft limit in of 100k. We should definitely increase this in the future